packages/experience-builder-sdk/src/communication/MouseOverHandler.ts

```diff
@@ -5,24 +5,36 @@ import { sendMessage } from './sendMessage';
 export class MouseOverHandler {
   private currentHoveredElementId: string | null = null;
 
+  private getMargins = (element: HTMLElement) => {
+    const styles = window.getComputedStyle(element);
+    const top = parseInt(styles.marginTop);
```

Comment:
We can expect here not only pixel values but also a variety of others. I assume this breaks this logic

Reply:
`/^\d{1,}(px|cm|mm|in|pt|pc|em|ex|ch|rem|vw|vh|vmin|vmax|%)?$/`

Reply:
As discussed via DM, we [get always pixel values here](https://stackoverflow.com/questions/48016082/window-getcomputedstyle-always-returns-measurement-in-pixels)